src/OrchardCore.Modules/OrchardCore.Deployment/Controllers/ImportController.cs

```diff
@@ -91,6 +92,10 @@ public async Task<IActionResult> Import(IFormFile importedPackage)
 
                     await _notifier.SuccessAsync(H["Deployment package imported."]);
                 }
+                catch (Exception ex)
+                {
+                    await _notifier.ErrorAsync(H[ex.Message]);
```

Comment:
Hi @MikeAlhayek ，Can you help me see what's wrong with my localization? Thank you!

Reply:
you can't do that. the use of H and S must be a literal string since this has to be extracted.

Reply:
But the notifier needs a `HtmlLocalizedString` ... What should I do?

Reply:
Maybe I can send the notify in Step logic?

Reply:
Is that okay?

```c#
                        var localizerString = S["Unable to add content-part to the '{0}' content-type. The part name cannot be null or empty.", type.Name];
                        await _notifier.ErrorAsync(H[localizerString]);
                        throw new InvalidOperationException(localizerString);
```

Reply:
Added a layer of exception catching message alerts, as there is currently no exception handling when a recipe is imported.



> you can't do that. the use of H and S must be a literal string since this has to be extracted. 

Since the string is already processed internally, no additional processing should be required

Reply:
I don't think you need to write the exception to the user using notifier.

Reply:
> I don't think you need to write the exception to the user using notifier.

I'm not sure I agree with this. Assuming that the person who has access to import recipes has full administrative access to the system, (I can't actually guess what other person roles have access to import recipes) 
it makes sense to throw some underlying exceptions directly to the user, as most Windows applications do, so that the user can view the exceptions directly through the page rather than having to log in to the server to view the logs. 
(I've developed a special feature to view log files online for this purpose~) 

Currently our recipe import page does nothing, which would make the user feel that the system is very unreliable After all, we can't anticipate all exceptions

https://github.com/OrchardCMS/OrchardCore/issues/15978

Reply:
You can submit another PR to address another issue. I suggest keeping this PR simple and doing that type check without worrying about exception handling.

Reply:
Okay,  got it. Thanks.